### PR TITLE
Fixes #239: changed all the files to PEP8 Compliant

### LINKED ISSUE #239: Files are not PEP8 Compliant
**I'm submitting a ...**
- [x] bug report

**Expected behavior:**
All Python files shuould be PEP 8 compliants

**Yes, I would like to work on the issue**

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #239 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -22,161 +23,165 @@
...
+        else:
+            flash('Please select a PNG image to Upload!', 'error')
+            return redirect(url_for('index'))
+    else:
+        if filename.find("png.csv") != -1:
+            if img == '':
+                flash('Please upload an image in \'PNG\' format!', 'error')
+                return redirect(url_for('index'))
+        else:
+            flash('Please upload a file in \'CSV\' format! Each line is a badge and the lines on the badge are separated by commas. Try it out!', 'error')
```
[the remark this answers is not in the document]
Comment (vaibhavsingh97) [reply app/main.py]: To fix this issue, we can use `\` or we can ignore `E501`

File: app/main.py [hunk 2]
```diff
@@ -22,161 +23,165 @@
...
 
-def generate_badges(_zip=False,_pdf=False):
-	if _zip == True and _pdf == True:
-		os.system('python ' + APP_ROOT + '/merge_badges.py -z -p')
-	elif _zip == True and _pdf == False:
-		os.system('python ' + APP_ROOT + '/merge_badges.py -z')
-	else:
-		os.system('python ' + APP_ROOT + '/merge_badges.py -p')
+def generate_badges(_zip=False, _pdf=False):
+    if _zip == True and _pdf == True:
```
[the remark this answers is not in the document]
Comment (vaibhavsingh97) [reply app/main.py]: Both statements are true so we can use
```python
if _zip and _pdf:
    pass
```

File: app/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jajodiaraghav) [reply app/main.py]: I think we should increase the line length for hound-ci to 150

**Comment by jajodiaraghav** [comment]
Looks good!

File: app/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vaibhavsingh97) [reply app/main.py]: Sure will update this PR

**Pull request closed without merging**